server/store/migrations/000003_add_survey_fields.up.sql

```diff
@@ -0,0 +1,2 @@
+{{- /* addColumnIfNeeded tableName columnName datatype constraint */ -}}
```

Comment:
Is this needed? I understand it is documentation, but 0/5 if this is the place for it.

Reply:
Nah, not needed, its just a habit from Focalboard days 😛 I've removed it now.